library/koji_tag_inheritance.py

```diff
@@ -103,34 +109,37 @@ def get_ids_and_inheritance(session, child_tag, parent_tag):
     return (child_id, parent_id, current_inheritance)
 
 
-def generate_new_rule(child_id, parent_tag, parent_id, priority):
+def generate_new_rule(child_id, parent_tag, parent_id, priority, maxdepth):
     """
     Return a full inheritance rule to add for this child tag.
 
     :param int child_id: Koji tag id
     :param str parent_tag: Koji tag name
     :param int parent_id: Koji tag id
     :param int priority: Priority of this parent for this child
+    :param int maxdepth: Max depth of the inheritance
```

Comment:
Looks like this can be either an `int` or `None`, right? You can add a comment like`(Or None to remove the limit)`